<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>motorTest1Class</class>
 <widget class="QMainWindow" name="motorTest1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>motorTest1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>端口设置</string>
    </property>
    <widget class="QPushButton" name="btnOpenSerialPort">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>打开端口</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comBoxMotor1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comBoxMotor2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>电机1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>电机2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cboxIsLink">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>是否联动</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblProtState">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>82</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(127, 127, 127);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comBoxForce">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>力传感器</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>电机1运动参数</string>
    </property>
    <widget class="QSpinBox" name="spboxMotor1Position">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-9999</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStartMotor1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>运行</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>行程</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spboxMotor1Vel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>速度</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGoZeroMotor1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>回零</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>电机2运动参数</string>
    </property>
    <widget class="QSpinBox" name="spboxMotor2Position">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-9999</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStartMotor2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>运行</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>行程</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spboxMotor2Vel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>速度</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGoZeroMotor2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>回零</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>力传感器设置</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>g</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStartReadForce">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开始读数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>测量值</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblForceVal">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>测量值</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearForce">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清    零</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblReadForceState">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(127, 127, 127);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>滤波器设置</string>
    </property>
    <widget class="QPushButton" name="btnSetFilter">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设置滤波器</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spboxFilterStrength">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comBoxFilterType">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>无滤波器</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>平均值滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中位值滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>一阶滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>滑动平均滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中位值平均滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>滑动中位值平均滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>平均值滤波+一阶滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中位值滤波+一阶滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>滑动平均滤波+一阶滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中位值平均滤波+一阶滤波</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>滤波强度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>滤波类型</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="motorTest1.qrc"/>
 </resources>
 <connections/>
</ui>
